shell/i18n/nl.i18n.json

```diff
@@ -0,0 +1,1787 @@
+          "explanation": "Voor elke vriend die een deellink aanmaakt krijgt u 2 GB"
+        },
+        "isNotPaid": {
+          "title": "Krijg een onbeperkte hoeveelheid korrels &amp; tot wel <span class=\"reward-amount\">2 GB</span> aan gratis opslagruimte wanneer uw vrienden delen op Oasis!",
+          "explanation": "Voor elke vriend die een deellink aanmaakt krijgt u 50 MB en daarnaast een onbeperkte hoeveelheid korrels."
+        }
+      },
+      "howTo": {
+        "title": "Delen is verwijzen",
+        "explanation": "Om te beginnen, open een korrel en klik op Toegang delen.",
+        "getStart": {
+          "step1": "U deelt een korrel met een vriend (die nog geen Oasis gebruiker is).",
+          "step2": "Uw vriend logt in.",
+          "step3": "Uw vriend deelt een korrel met een ander (waaronder uzelf)."
+        }
+      },
+      "status": {
+        "title": "Uw verwijzingen & status",
+        "explanation": "Wanneer u een deel link verzend verschijnen mensen hieronder wanneer ze inloggen.",
+        "pending": {
+          "title": "Moet nog delen.",
+          "explanation": "Vertel deze mensen om een korrel aan te maken en die met u te delen. U moet de korrel ook bekijken!"
+        },
+        "success": {
+          "title": "Succesvolle verwijzingen",
+          "explanation": "U heeft deze mensen met succes verwezen, gefeliciteerd!"
+        }
+      },
+      "finePrint": {
+        "title": "De kleine lettertjes",
+        "warningWithAccountLink": "Uw verwijzingsbonus is voor 1 jaar na de verwijzing geldig. De voorwaarden van dit verwijzingsprogramma kunnen veranderen zonder waarschuwing, bonussen die u al heeft ontvangen worden daar niet door beïnvloed. Als u nog meer ruimte nodig heeft, <a href=\"%s\">upgrade dan uw account.</a>",
+        "explanation": "Betaalde abonnees kunnen tot 30 GB gratis opslagruimte krijgen, 2 GB per persoon die ze verwijzen, naast de hoeveelheid opslag van hun account niveau. Eigenaren van gratis accounts kunnen een onbeperkte hoeveelheid korrels en 50MB opslagruimte per persoon die ze verwijzen verdienen, tot in totaal 2GB."
+      }
+    },
+    "about": {
+      "title": "Over Sandstorm",
+      "intro": {
+        "explanation": "<a href=\"https://sandstorm.io\">Sandstorm.io</a> is een open source hosting platform voor persoonlijke/privé instanties van web apps. Gebruikers kunnen software uploaden en installeren via een simpele app-store achtige interface. Naast betere privacy <a href=\"https://blog.sandstorm.io/news/2014-07-21-open-source-web-apps-require-federated-hosting.html\">is dit de enige manier om Open Source web apps haalbaar te maken.</a>",
+        "version": "Versie: %s",
+        "unofficialWarning": "<b>WAARSCHUWING:</b> deze server draait een onofficiële build van Sandstorm die aanpassingen kan bevatten die niet zijn gecontroleerd door de ontwikkelaars."
```

Comment:
It's in the existing localizations too, so orthoganal to this pr, but we should probably not be using the `<b>` tag. @kentonv, maybe we should change this to `<strong>`?

Reply:
I don't have a strong opinion (... so to speak), but would accept a pull request.

Reply:
I'll get on it